kubectl-plugin/kubectl-waiops

```diff
@@ -1178,10 +1208,12 @@ then
     # if statement block to account for versions over 4.2 (clusterwide changes)
     if [ "${CLUSTER_SCOPE_INSTALL}" == "true" ];
     then
-        if [[ "${VERSION_AIOPSORCHESTRATOR}" != "3.3" ]] || [[ "${VERSION_AIOPSORCHESTRATOR}" != "3.4" ]] || [[ "${VERSION_AIOPSORCHESTRATOR}" != "3.5" ]] || [[ "${VERSION_AIOPSORCHESTRATOR}" != "3.6" ]] || [[ "${VERSION_AIOPSORCHESTRATOR}" != "3.7" ]] || [[ "${VERSION_AIOPSORCHESTRATOR}" != "4.0" ]] || [[ "${VERSION_AIOPSORCHESTRATOR}" != "4.1" ]];
+        getSubscriptionStatusGrep "ibm-automation-elastic" "$INSTALLATION_NAMESPACE" 
+        if [[ "${VERSION_AIOPSORCHESTRATOR}" != "4.7" ]];
         then
-            getSubscriptionStatusGrep "ibm-automation-elastic" "$INSTALLATION_NAMESPACE" 
```

Comment:
Any reason Elastic was pulled out of this `if`? Did we add this subscription with the removal of IAF in 4.1?